config/environment.js

```diff
@@ -59,12 +59,6 @@ module.exports = function (environment) {
       completeExistingMessages: true,
       showFileInfo: true,
     },
-    'ember-a11y-testing': {
```

Comment:
Oh no. If this had to get pulled as well I'm not sure I'm ok with that. The tests we have were difficult to get working and removing them is taking a big step back as some components which are currently accessible will probably get broken.

Reply:
alright, i backed out of that.